benefits/eligibility/templates/eligibility/includes/media-item--idcardcheck--start--veteran.html

```diff
@@ -3,24 +3,25 @@
 {% load i18n %}
 
 {% block heading %}
-  {% translate "Access to your preferred VA related account" %}
+    {% translate "Access to your preferred VA related account" %}
 {% endblock heading %}
 
 {% block body %}
-  <div class="media-body--details">
-    <p>
-      {% translate "You will need to sign into VA.gov in order to verify your identity and your veteran status. VA.gov uses the following services:" %}
-    </p>
-    <div class="media-body--items">
-      <ul>
-        <li>{% translate "Login.gov" %}</li>
-        <li>{% translate "ID.me" %}</li>
-        <li>{% translate "DS Logon" %}</li>
-        <li>{% translate "My HealtheVet" %}</li>
-      </ul>
-      <p class="pt-1">
-        {% translate "If you do not have an account with any of these services, you will need to create one. We recommend using Login.gov." %}
-      </p>
+    <div class="media-body--details">
+        <p>
+            {% translate "You will be able to create an account using your email address if you do not already have one. We use your Login.gov account to verify your identity." %}
+            {% translate "Learn more about identity verification" as text %}
+            {% include "core/includes/modal-trigger.html" with modal="modal--identity-verification" text=text period=True %}
+        </p>
+        <div class="media-body--items">
+            <p>{% translate "For this process you will need:" %}</p>
+            <ul>
+                <li>{% translate "Your state-issued ID card" %}</li>
+                <li>{% translate "Your Social Security number" %}</li>
+                <li>{% translate "A phone number with a phone plan associated with your name" %}</li>
+            </ul>
+        </div>
     </div>
-  </div>
+
+    {% include "eligibility/includes/modal--senior-start-help.html" with size="modal-lg" id="modal--identity-verification" header="p-md-2 p-3" body="pb-md-3 mb-md-3 mx-md-3 py-0 pt-0 absolute-top" %}
```

Comment:
This is problematic. We need to either rename this include or create a new version for veterans, we can't use `senior` here, too confusing.

Since we've been using the eligibility type as the name, I'd recommend creating a new version for `veterans` vs. renaming to `login-gov` or something.

Reply:
I vote for renaming to `login-gov`